ajax/libs/es7-shim/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "es7-shim",
+  "description": "ECMAScript 7 compatibility shims for legacy JavaScript engines",
+  "homepage": "http://github.com/es-shims/es7-shim/",
+  "author": "Jordan Harband <ljharb@gmail.com> (https://github.com/ljharb/)",
+  "license": "MIT",
+  "filename": "es7-shim.min.js",
+  "npmName": "es7-shim",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "es7-shim.+(js|min.js)"
```

Comment:
We can host all the files under `dist`
So let modify from `"es7-shim.+(js|min.js)"` to `**/*`.
Thank you.